lib/icinga/notification.cpp

```diff
@@ -439,6 +448,20 @@ void Notification::BeginExecuteNotification(NotificationType type, const CheckRe
 			}
 		}
 
+		if (type == NotificationProblem && !reminder && !checkable->GetVolatile()) {
+			auto [host, service] = GetHostService(checkable);
+			uint_fast8_t state = service ? service->GetState() : host->GetState();
+
+			if (state == (uint_fast8_t)GetLastNotifiedStateByUser()->Get(userName)) {
+				Log(LogNotice, "Notification")
+					<< "Notification object '" << notificationName << "': We already notified user '" << userName << "' for a "
+					<< (service ? NotificationServiceStateToString(service->GetState()) : NotificationHostStateToString(host->GetState()))
+					<< " problem. Likely the last problem-to-problem state change was filtered out. Not sending duplicate notification.";
```

Comment:
This message looks like this when it is being logged, and doesn't seem to be very informative IMHO.
```bash
[2023-11-22 10:55:46 +0100] notice/Notification: Notification object 'icinga2!test!notify': We already notified user 'yhabteab' for a Critical problem. Likely the last problem-to-problem state change was filtered out. Not sending duplicate notification.
```

Why isn't it rephrased like the existing logs above it? Specifically, I didn't understand what this is supposed to tell a user who doesn't know its implementation details.
```
 Likely the last problem-to-problem state change was filtered out. Not sending duplicate notification.
```